charts/nginx/templates/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "nginx.fullname" . }}
+  labels:
+    app: {{ template "nginx.name" . }}
+    chart: {{ template "nginx.chart" . }}
+    release: {{ .Release.Name }}
+    instance: {{ .Values.Instance | quote }}
+    instanceID: {{ .Values.SLATE.Instance.ID | quote  }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "nginx.name" . }}
+      instance: {{ .Values.Instance | quote }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "nginx.name" . }}
+        chart: {{ template "nginx.chart" . }}
+        instance: {{ .Values.Instance | quote }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+      - name: nginx
+        image: "nginx:1.15.9"
```

Comment:
@LincolnBryant last week you mentioned we should try for the `latest` tags for base images correct?

Reply:
This is the same as appVersion. The chart currently has a appVersion of "1.15.9" and thus the image tag is 1.15.9. 
Any objections to update appVersion to 1.23.0 and also make image tag configurable from the helm values(add image: tag)? Although this would make the helm appversion inconsistent with what's running if an instance has the image tag set.

Reply:
It would be worthwhile to hear Lincoln's thinking on this, although again I would like to offer a note on how we reasoned about this before:
Having the version pinned in the chart is about the only way that SLATE can make any meaningful statement about the security of what the chart will install. `latest` can change meaning at any time, so there's not even any guarantee that multiple instances started from the same chart version at slightly different times would be running the same application version. Likewise, we chose to forbid users selecting image tags because this would open up uncertainty about what charts would install. That particularly matters for more-highly-security-conscious sites, who are only willing to allow SLATE and some or all of its users in if they can know exactly what those users will be able to run at any given time.

Reply:
So, I haven't totally followed the thread here but my thinking on this was as such... We do not allow **images** to be selected at instantiation time (e.g., values.conf), but we DO allow **image tags**. So, for nginx, that would look something like:

`image: nginx:{{ tag }}`

Where `image: nginx` is fixed, but `{{ tag }}` should be dependent on the values file. If there's a known bad version upstream, that's upstream's problem to take care of by deleting that tag or otherwise fixing it.

I am of the opinion that we should preferentially default to `latest` tag _or_ whatever scheme the upstream image provider chooses for their most up-to-date chart. 

This way, we ensure that SLATE provides, by default, the latest & greatest for any container we don't build ourselves. Users can then _choose_ to use some other older/more stable/etc tag at their own discretion. I feel like this best insulates us from issues where SLATE is shipping an ancient version of some application that is known to be vulnerable, but also allows the user to make a conscious choice to use some particular release. IMO, it is better to ship a potentially broken (configuration-wise) but ostensibly up to date app, than a well-known vulnerable app. 

That said, if we chose to stick to particular tagged releases of apps, then we need to commit to, e.g., a weekly refresh all applications in the stable catalog. It's important to keep on top of this - if that's a task the team wants to take on then I'm all for it.